code/datums/cache/crew.dm

```diff
@@ -1,7 +1,14 @@
 GLOBAL_DATUM_INIT(crew_repository, /datum/repository/crew, new())
 
+/datum/repository/crew
+	var/static/list/heads
+	var/static/list/ert_jobs
+
 /datum/repository/crew/New()
 	cache_data = list()
+	// These jobs are used to highlight those with a command ID. Stolen from nttc.dm
+	heads = heads || list("Captain", "Head of Personnel", "Nanotrasen Representative", "Blueshield", "Chief Engineer", "Chief Medical Officer", "Research Director", "Head of Security", "Magistrate", "AI")
+	ert_jobs = ert_jobs || list("Emergency Response Team Officer", "Emergency Response Team Engineer", "Emergency Response Team Medic", "Emergency Response Team Leader", "Emergency Response Team Member")
```

Comment:
2 things.

Why the `heads = heads || list` operator, and why not just have a list of people to bold, instead of 2 separate lists, and why are you including `AI` when they dont have sensors

Reply:
Ah, I thought there'd be multiple instances of `/datum/repository/crew` (which wouldn't make sense) so I wanted to define the lists only once. I'll fix that.

As for the bolding and AI part, I kinda mindlessly took the code from Telecomms and forgot to adjust it